.github/actions/conventional-commits/action.yml

```diff
@@ -1,20 +1,78 @@
 name: Conventional Commits Checking
 description: "Validates that a pull request title is compatible with Conventional Commits."
+inputs:
+  enforce_identifier_prefixes:
+    description: "List of JIRA (or similar) project identifiers to enforce. eg: `feat(FOO-123):...` or `feat(BAR-123):...` then pass in FOO\nBAR"
+    required: false
 runs:
   using: composite
   steps:
+    - uses: actions/setup-node@v3
+      with:
+        node-version: "18"
+        check-latest: true
+    - run: npm install csv-parse
+      shell: bash
     - uses: actions/github-script@v5
       with:
         script: |
-          const validator = /^(chore|feat|fix|revert|docs|style)(\([a-z ]+\))?(!)?: (.)+$/
+          const {parse} = require('csv-parse/sync');
```

Comment:
If we are going to be using a library, is there just a node library that checks this already? all this regex stuff seems very error prone, but you know my feelings about regexp.

Reply:
like this? https://github.com/conventional-changelog/commitlint

Reply:
you're right